composer.json

```
@@ -17,10 +17,10 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.4 || ^8.0",
```

Looks like PHPUnit 10 is going to only support PHP 8 and above, so I guess we'll want to mirror that. 👍🏻 (Same on the workflows).
```suggestion
        "php": "^8.0",
```

---

Nice! Now we have reason for this. Ahah.